IntelliTect.TestTools.Selenate.Tests/ElementHandlerTests.cs

```diff
@@ -0,0 +1,338 @@
+using Moq;
+using OpenQA.Selenium;
+using System;
+using Xunit;
+
+namespace IntelliTect.TestTools.Selenate.Tests
+{
+    public class ElementHandlerTests
+    {
+        [Fact]
+        public void ClickDoesNotThrowIfSuccessful()
```

Comment:
nit: In general (not a hard and fast rule) I prefer to avoid negatives in test names as it tends to leave things pretty open ended; and instead phrase it in terms of the positive expectation.
```suggestion
        public void ClickDoesIsInvokedOnElementWhenFound()
```

Reply:
Done!